.php-cs-fixer.cache

```diff
@@ -0,0 +1 @@
rt_slash":true,"no_whitespace_in_blank_line":true,"ordered_class_elements":{"order":["use_trait"]},"return_type_declaration":true,"short_scalar_cast":true,"single_trait_insert_per_statement":true,"ternary_operator_spaces":true,"visibility_required":true,"blank_line_after_namespace":true,"constant_case":true,"control_structure_braces":true,"control_structure_continuation_position":true,"elseif":true,"function_declaration":true,"indentation_type":true,"line_ending":true,"lowercase_keywords":true,"no_break_comment":true,"no_closing_tag":true,"no_multiple_statements_per_line":true,"no_space_around_double_colon":true,"no_spaces_after_function_name":true,"no_trailing_whitespace":true,"no_trailing_whitespace_in_comment":true,"single_blank_line_at_eof":true,"single_line_after_imports":true,"spaces_inside_parentheses":true,"statement_indentation":true,"switch_case_semicolon_to_colon":true,"switch_case_space":true,"encoding":true,"full_opening_tag":true,"array_push":true,"combine_nested_dirname":true,"dir_constant":true,"ereg_to_preg":true,"error_suppression":true,"fopen_flag_order":true,"fopen_flags":{"b_mode":false},"function_to_constant":true,"get_class_to_class_keyword":true,"implode_call":true,"is_null":true,"logical_operators":true,"modernize_strpos":true,"modernize_types_casting":true,"native_constant_invocation":{"strict":false},"native_function_invocation":{"include":["@compiler_optimized"],"scope":"namespaced","strict":true},"no_alias_functions":true,"no_homoglyph_names":true,"no_php4_constructor":true,"no_unneeded_final_method":true,"no_useless_sprintf":true,"non_printable_character":true,"ordered_traits":true,"php_unit_construct":true,"php_unit_mock_short_will_return":true,"php_unit_set_up_tear_down_visibility":true,"php_unit_test_annotation":true,"psr_autoloading":true,"self_accessor":true,"set_type_to_cast":true,"string_length_to_empty":true,"string_line_ending":true,"ternary_to_elvis_operator":true,"pow_to_exponentiation":true,"no_trailing_whitespace_in_string":true,"header_comment":{"header":"This file is part of the OverblogDataLoaderBundle package.\n\n(c) Overblog <http:\/\/github.com\/overblog\/>\n\nFor the full copyright and license information, please view the LICENSE\nfile that was distributed with this source code."}},"hashes":{"tests\/Functional\/app\/AppKernel.php":"f0815cab559ee2b5d64e42e61df9932a","tests\/Functional\/app\/UserDataProvider.php":"5c14d46d5b5ad419196d30e1774c7755","tests\/Functional\/BootTest.php":"08e2116f57108c1b8aa732e5feac6f5a","tests\/Functional\/TestCase.php":"8a3a57fff184a0f7f21c5be51c844518","tests\/Functional\/UserLoaderTest.php":"148f54513cf4fb7672cb745866e130c7","tests\/DependencyInjection\/OverblogDataLoaderExtensionTest.php":"2da2a2c5d38c29bd6f1cffde63cc9527","src\/OverblogDataLoaderBundle.php":"23e6aa8cce72b417398a4b81e14418c0","src\/DependencyInjection\/OverblogDataLoaderExtension.php":"38076a63da3a07d0e2c4b1ced3283bbe","src\/DependencyInjection\/Configuration.php":"21fdd2d6296d9715a44429f0d0bc3c50"}}
```

Comment:
Can you please delete this and add it to gitignore file please?

Reply:
Oops, done.